backend/src/spotify.py

```diff
@@ -94,8 +94,10 @@ def refresh_access_token(self, refresh_token):
         user_info = self.get_current_user(access_token)
 
         resp = make_response()
-        resp.set_cookie("spotify_access_token", access_token)
-        resp.set_cookie("user_id", user_info.id)
+        resp.set_cookie(
+            "spotify_access_token", access_token, samesite="None", secure=True
```

Comment:
(Not important)
I wonder if this would be worth extracting into a method somewhere, so that you don't need to remember to specify samesite every time - this feels like the sort of issue that would be bound to catch out a new dev who adds a cookie that then works locally but not when deployed!

Reply:
Great shout. Will do so!